src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Shared/NavMenu.CallsMicrosoftGraph.razor

```diff
@@ -1,33 +1,33 @@
 ﻿<div class="top-row pl-4 navbar navbar-dark">
     <a class="navbar-brand" href="">BlazorServerWeb-CSharp</a>
-    <button class="navbar-toggler" @onclick="ToggleNavMenu">
+    <button title="Navigation menu" class="navbar-toggler" @onclick="ToggleNavMenu">
         <span class="navbar-toggler-icon"></span>
     </button>
 </div>
 
 <div class="@NavMenuCssClass" @onclick="ToggleNavMenu">
-    <ul class="nav flex-column">
-        <li class="nav-item px-3">
+    <nav class="flex-column">
```

Comment:
Changing from `ul` to `nav` is valuable because:

 * If it's `ul`, then the reader will announce things like *List with three items*. But that's kind of weird and useless, since you don't know what it's a list containing. Also there's no natural way to get here using a keyboard shortcut.
 * If it's `nav`, the screen reader will say things like *Navigation landmark* instead, and you can jump right to it from anywhere else by pressing the "navigation" keyboard shortcut (usually `d` or `shift+d`).

I decided to put the `nav` wrapper here instead of around the whole `NavMenu` because really the `NavMenu` is more of a complete "sidebar" that might commonly be extended to include non-navigation-related content too.